README.md

```diff
@@ -9,7 +9,7 @@ I am Kaiwalya Koparkar, Currently I am working as Program manager at WeMakeDevs,
 [![Website Badge](https://img.shields.io/badge/-Portfolio-black?style=flat-square&logo=Wordpress&logoColor=white&link=https://kaiwalyakoparkar.github.io/)](https://kaiwalyakoparkar.github.io/)
 [![Youtube Badge](https://img.shields.io/badge/-Kaiwalya%20Koparkar-darkred?style=flat-square&logo=youtube&logoColor=white&link=https://www.youtube.com/channel/UCZow8pOHiyz26yl4Da-Mfzw)](https://www.youtube.com/channel/UCZow8pOHiyz26yl4Da-Mfzw)
 
-<img src="https://activity-graph.herokuapp.com/graph?username=kaiwalyakoparkar&bg_color=0f2d3d&color=1cadfb&line=1cadfb&point=1cadfb&area=true&hide_border=true">
+[![Ashutosh's github activity graph](https://github-readme-activity-graph.cyclic.app/graph?username=kaiwalyakoparkar&theme=dracula)](https://github.com/ashutosh00710/github-readme-activity-graph)
```

Comment:
Change `Ahutosh's` to `Kaiwalya's` also please change the color of the graph

Reply:
What color should it be changed to?